components/Footer/Footer.module.scss

```diff
@@ -4,19 +4,17 @@
   }
 }
 .footerNav {
-  width: calc(100% - 30px);
```

Comment:
Remove the scroll

Reply:
Removed

Reply:
Add spacing to the bottom to be equal to the horizontal spacing

Reply:
Done

Reply:
Good job